### ISSUE TITLE
session request should respect timeout

### ISSUE THREAD

**Issue by balmas** [alpheios-project/alpheios_nemo_ui#306]
the request for the session should fail if the token is timed out

**Comment by balmas** [comment]
this can be tested in the texts-test environment now using Alpheios Embedded Library 3.1.4.45

@monzug for ease of the initial testing, the test environment is set to use a 1 minute timeout on the login.  Please verify that you get the timeout message and are logged out after 1 minute on the desktop and mobile.

Let me know if that test succeeds for you and if so, I'll switch back to the live services with the 24 hour timeout for a final test. Thanks!

**Comment by monzug** [comment]
message generated in mobile and in text env in Chrome. however after the
time out if I click on log in it does let me in without prompting for id/psw

**Comment by balmas** [comment]
>however after the time out if I click on log in it does let me in without prompting for id/psw

yes that's expected with the dev setup, since the auth0 login cookie isn't really timed out, we're just pretending it is.

I will reset the environment to the production services with the 24 hours timeout.

**Comment by monzug** [comment]
perfect.

**Comment by balmas** [comment]
ok test environment is now back to the 24 hours timeout and production services.

**Comment by monzug** [comment]
24 hrs test  in text env is on

**Comment by monzug** [comment]
email on Nov 21:
confirm that in mobile I didn't get any message nor I was logged out after the 24 hrs session.
in text env on Chrome, the session has timed out message is generated.  however it shows the toolbar with the highlighted wordlist and when on user account, user is still logged in. need to click on log out, to start a new session.
there are also screenshot of text env.
email from Nov 25:
I know we haven't fully solved the timeout issues in the text environment.
giving back to Bridget to see if we want to implement a fix or if this is ok as is

**Comment by balmas** [comment]
yes we do, eventually. So it should remain open.  Targeting for our next incremental for now.
